docs/BackupAndRestore.md

```diff
@@ -168,7 +136,29 @@ A remote _stream_ source is downloaded and extracted directly into the archive d
 - _backupset_ - Hot copy backup set which is either fetched from a remote source or available in the backup directory.
 A remote _backupset_ requires additional space as it is downloaded and extracted temporarily in the archive volume to be used during archive restore operation.
 
-### Distributed database in-place restore
+### Fine-grained archive selection
+
+NuoDB _restore_ chart provides several ways to select which archives should be restored during the [distributed database restore](#distributed-database-restore-from-source) or [archive seed restore](#archive-seed-restore).
```

Comment:
I suspect that both "distributed database restore" and "archive seed restore" are names and as such should be capitilized